python/ray/tests/test_global_state.py

```diff
@@ -59,7 +59,7 @@ def cpu_task():
 @pytest.mark.skipif(
     pytest_timeout is None,
     reason="Timeout package not installed; skipping test that may hang.")
-@pytest.mark.timeout(20)
+@pytest.mark.timeout(120)
```

Comment:
Why do we have really long timeout here?

Reply:
It looked like the test was timing out since it starts and stops a lot of processes.